Gems/Multiplayer/Code/Source/MultiplayerStatSystemComponent.h

```diff
@@ -58,8 +58,11 @@ namespace Multiplayer
 
         struct CumulativeAverage
         {
+            using Type = AZ::IO::AverageWindow<double, double, AZ::IO::s_statisticsWindowSize>;
```

Comment:
I'm not sure how I feel about Type as a name.

Reply:
Fair, updated.